js/workersrc.js

```diff
@@ -15,3 +15,7 @@ redirectIfNotDisplayedInFrame();
 PDFJS.openExternalLinksInNewWindow = true;
 PDFJS.isEvalSupported = false;
 PDFJS.workerSrc = document.getElementsByTagName('head')[0].getAttribute('data-workersrc');
+
+PDFViewerApplication.download = function() {
+	window.open(decodeURIComponent(window.location.search.substr(6)), '_blank');
```

Comment:
For me this does not download the file, but open yet another new tab where you then have to click the "Download button". Then you finally get the file